docs/03-tutorials/00-api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,304 @@
+  apiVersion: dns.gardener.cloud/v1alpha1
+  kind: DNSEntry
+  metadata:
+    name: dns-entry
+    namespace: $NAMESPACE
+    annotations:
+      dns.gardener.cloud/class: garden
+  spec:
+    dnsName: "$SUBDOMAIN"
+    ttl: 600
+    targets:
+      - $IP
+  EOF
+  ```
+
+  </details>
+  <details>
+  <summary>
+  Non-Gardener cluster
+  </summary>
+
+  ```bash
+  cat <<EOF | kubectl apply -f -
+  apiVersion: dns.gardener.cloud/v1alpha1
+  kind: DNSEntry
+  metadata:
+    name: dns-entry
+    namespace: $NAMESPACE
+  spec:
+    dnsName: "$SUBDOMAIN"
+    ttl: 600
+    targets:
+      - $IP
+  EOF
+  ```
+
+  </details>
+  </div>
+
+  >**NOTE:** To check the Ingress Gateway IP, run:
+  >```bash
+  >kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+  >```
+
+  >**NOTE:** You can create many DNSEntry CRs for one DNSProvider, depending on the number of subdomains you want to use. To simplify your setup, consider using a wildcard subdomain if all your DNSEntry objects share the same subdomain and resolve to the same IP, for example: *.api.mydomain.com.
+
+5. Create an Issuer CR. Export values of the **metadata.spec.acme.email**, **metadata.spec.domains.include**, and **metadata.spec.domains.exclude** parameters as environment variables. As the values for the **metadata.spec.domains.include** parameter, use the main domain, the subdomain from the **spec.dnsName** parameter of the DNSEntry CR, and a wildcard DNS record. Run:
+
+   ```bash
+   export EMAIL={YOUR_EMAIL_ADDRESS}
+   export DOMAIN={CLUSTER_DOMAIN}
+   export SUBDOMAIN={YOUR_SUBDOMAIN}
+   export WILDCARD={YOUR_WILDCARD_SUBDOMAIN} #e.g. *api.mydomain.com
+   ```
+
+   Create an Issuer CR. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: cert.gardener.cloud/v1alpha1
+   kind: Issuer
+   metadata:
+     name: letsencrypt-staging
+     namespace: default
+   spec:
+     acme:
+       server: https://acme-staging-v02.api.letsencrypt.org/directory
+       email: $EMAIL
+       autoRegistration: true
+       # Name of the Secret used to store the ACME account private key
+       # If doesn't exist, a new one is created
+       privateKeySecretRef:
+         name: letsencrypt-staging-secret
+         namespace: default
+       domains:
+         include:
+           - $DOMAIN
+           - $SUBDOMAIN
+           - "$WILDCARD"
+         # Optionally, restrict domain ranges for which certificates can be requested
+   #     exclude:
+   #       - my.sub2.mydomain.com # Edit this value
+   EOF
+   ```
+
+6. Create a Certificate CR. Export values of the **metadata.namespace**, **spec.secretName**, **spec.commonName**, and **spec.issuerRef.name**, and **spec.dnsNames** parameters as environment variables. As the value for the **spec.issuerRef.name** parameter, use the value from te **metadata.name** parameter of the Issuer CR. Run:
+
+   ```bash
+   export SECRET_2={SECRET_NAME} #e.g: httpbin-tls-credentials
```

Comment:
It is not clear to me if this secret should exist or will be created. And why `SECRET_2` name?